### ISSUE TITLE
Script to generate audio

### ISSUE THREAD

**Issue by arijit17** [deNsuh/segan-pytorch#15]
It will be good to have a separate script just to generate audio from a trained model.
Example, you feed in a noisy signal to a trained generator and you get an enhanced signal.

**Comment by dansuh17** [comment]
Of course. There is a plan on the way of implementing 1. save / load pretrained model 2. generate using the model. Please hold on :)

**Comment by arijit17** [comment]
Great 👍 
1. Is for Transfer Learning I believe.
2. Is more urgent 
Thanks for the nice work!
